pkg/client/daemon/tunrouter.go

```diff
@@ -83,6 +84,15 @@ type tunRouter struct {
 	// dnsLocalAddr is the address of the local DNS server
 	dnsLocalAddr *net.UDPAddr
 
+	// Cluster subnets reported by the traffic-manager
+	clusterSubnets []*net.IPNet
+
+	// Subnets configured by the user
+	alsoProxySubnets []*net.IPNet
+
+	// Subnets that the router is currently configured with
+	subnets []*net.IPNet
```

Comment:
Would `curSubnets` be a better name?

Reply:
Perhaps also the comment should say that it should *only* be used within `refreshSubnets()`?